pages/common/surfraw.md

```diff
@@ -0,0 +1,29 @@
+# surfraw
+
+> Surfraw is a CLI to query a variety of web search engines.
+> It consists of a collection of elvi, each of which knows how to search a specific website.
```

Comment:
Perhaps this will read better?
```suggestion
> CLI to query a variety of web search engines.
> Consists of a collection of elvi, each of which knows how to search a specific website.
```